### PR TITLE
(BUGZ-1112) Make 20:20 vision equal seeing 1m cube from 400 meters away

### PR DESCRIPTION
**Pull request by sabrina-shanman**
https://highfidelity.atlassian.net/browse/BUGZ-1112

This PR fixes incorrect default LOD and incorrect debug display values for current LOD. It also changes the internal representation of LOD in LODManager to be the half angular resolution, deprecates some LODManager script properties associated with the old representation, and does a bit of cleanup.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20818/HighFidelity-Beta-PR16049-49f1ecb.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20818/HighFidelity-Quest-Beta-PR16049-49f1ecb.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20818/HighFidelity-Beta-PR16049-49f1ecb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20818/HighFidelity-Beta-Interface-PR16049-49f1ecb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20820/HighFidelity-Beta-PR16049-07ea796.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20820/HighFidelity-Beta-Interface-PR16049-07ea796.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20820/HighFidelity-Beta-PR16049-07ea796.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20820/HighFidelity-Quest-Beta-PR16049-07ea796.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20820/HighFidelity-Beta-PR16049-07ea796.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20820/HighFidelity-Beta-Interface-PR16049-07ea796.exe

File: interface/src/LODManager.h [hunk 1]
```diff
@@ -59,6 +60,7 @@ class LODManager : public QObject, public Dependency {
 
         Q_PROPERTY(float lodQualityLevel READ getLODQualityLevel WRITE setLODQualityLevel NOTIFY lodQualityLevelChanged)
 
+        // Deprecated
```
Comment (samcake) [inline interface/src/LODManager.h]: i don;t understand this comment ?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20822/HighFidelity-Beta-PR16049-16593ec.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20822/HighFidelity-Beta-Interface-PR16049-16593ec.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20822/HighFidelity-Beta-PR16049-16593ec.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20822/HighFidelity-Quest-Beta-PR16049-16593ec.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20822/HighFidelity-Beta-PR16049-16593ec.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20822/HighFidelity-Beta-Interface-PR16049-16593ec.exe

**Pull request merged**